### PR TITLE
terence lin and shawn

### PR DESCRIPTION
**Pull request by Shawn-Sydney**

### CODE REVIEW

File: frontends/property/src/components/common/Carousel.js [hunk 1]
```diff
@@ -1,29 +1,49 @@
...
+	useEffect(() => {
+		fetch("http://localhost:3000/api/images/type/hotel")
+			.then((data) => data.json())
+			.then((data) => setImages(data));
+	}, []);
 
-    if (!images) return <h1>...Loading</h1>
+	if (!images) return <h1>...Loading</h1>;
+	const handleRadioClick = (e) => {
+		const deactive = document.getElementsByClassName("carousel-active");
```
Comment (jackietian) [inline frontends/property/src/components/common/Carousel.js:16]: this is jquery and native js ways of doing styles changes.

use react way pls. e.g conditional rendering.

File: frontends/property/src/components/Home.js [hunk 2]
```diff
@@ -1,15 +1,15 @@
-import Carousel from './common/Carousel'
-import BookingForm from './common/BookingForm'
-import Testimonial from './common/Testimonial'
+import Carousel from "./common/Carousel";
```
Comment (jackietian) [inline frontends/property/src/components/Home.js:1]: too many styles changes. I cannot focus on your changes. could you pls revert back the styles changes?

File: frontends/property/src/styles/carousel.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
+section.carousel {
```
Comment (jackietian) [inline frontends/property/src/styles/carousel.scss:1]: section will be reused in main.scss. so better to move it back to `main.scss`

**Review by jackietian — CHANGES_REQUESTED** [review]

**Comment by terence1988** [comment]
PR updated?

File: frontends/property/src/styles/carousel.scss (hunk 3 above)
Comment (terence1988) [reply frontends/property/src/styles/carousel.scss:1]: done

**Pull request closed without merging**